src/execute/oas3/parameter-builders.js

```diff
@@ -21,19 +21,23 @@ export function path({ req, value, parameter }) {
     escape: true,
   });
 
-  req.url = req.url.split(`{${name}}`).join(styledValue);
+  if (styledValue !== undefined) {
+    req.url = req.url.split(`{${name}}`).join(styledValue);
+  } else {
+    req.url = req.url.split(`{${name}}`).join('');
```

Comment:
Added this as otherwise URLs for undefined parameters would have a comma: `http://localhost:8080/,`, which I noticed when testing the fix.